docs-new/docs/installation/configuration/cert-manager.md

```diff
@@ -61,5 +61,34 @@ Note the following about these fields:
   must be `keptn-certs`.
 * Substitute the namespace placeholders with your namespace, where Keptn is installed.
 
-See the [CA Injector](https://cert-manager.io/docs/concepts/ca-injector/)
-documentation for more details.
+## Injecting CA Annotations
+
+`cert-manager.io` supports specific annotations for injectable resources, namely
+`cert-manager.io/inject-ca-from`, `cert-manager.io/inject-ca-from-secret`,
+or `cert-manager.io/inject-apiserver-ca`,
```

Comment:
Looks good! Let's please try to avoid naming the annotations here and rather link the cert-manager.io documentation (https://cert-manager.io/docs/concepts/ca-injector/). It will be easier to maintain in the future when something will change.

With this minor adaptation, I would say it's ready for merging :)

Reply:
updated

Reply:
Thank you, approved! Let's wait for a second review before merging it